infra/reachability-metadata/src/main/resources/META-INF/native-image/com.github.ben-manes.caffeine/caffeine/2.9.3/reflect-config.json

```diff
@@ -99,19 +99,16 @@
   "name":"com.github.benmanes.caffeine.cache.StripedBuffer",
   "fields":[{"name":"tableBusy"}]
 },
+{
+  "condition":{"typeReachable":"com.github.benmanes.caffeine.cache.NodeFactory"},
+  "name":"com.github.benmanes.caffeine.cache.PDWMS",
+  "methods":[{"name":"<init>","parameterTypes":[] }]
+},
 {
   "condition":{"typeReachable":"com.github.benmanes.caffeine.cache.StripedBuffer"},
   "name":"java.lang.Thread",
   "fields":[{"name":"threadLocalRandomProbe"}]
 },
-{
-  "condition":{"typeReachable":"com.github.benmanes.caffeine.cache.LocalLoadingCache"},
-  "name":"org.apache.shardingsphere.infra.parser.cache.SQLStatementCacheLoader"
-},
-{
-  "condition":{"typeReachable":"com.github.benmanes.caffeine.cache.LocalLoadingCache"},
-  "name":"org.apache.shardingsphere.sql.parser.core.database.cache.ParseTreeCacheLoader"
-},
```

Comment:
- I noticed that this was introduced via #32633. But the question is, these two JSONs are actually already located in `infra/reachability-metadata/src/main/resources/META-INF/native-image/org.apache.shardingsphere/generated-reachability-metadata/reflect-config.json`, does this affect any unit tests?